plugins/modules/zos_mount.py

```diff
@@ -0,0 +1,995 @@
+# Apache License, Version 2.0 (see https://opensource.org/licenses/Apache-2.0)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community',
+}
+
+DOCUMENTATION = r"""
+---
+module: zos_mount
+author:
+    - "Rich Parker (@richp405)"
+short_description: Mount a filesystem for Unix System Services (USS)
+description:
+  - zos_mount connects an existing, mountable file to USS.
+  - Mountable file needs a valid, unique Fully Qualified Name.
+  - Target folder will be created and/or unmounted if needed
+options:
+    path:
+        description:
+            - The absolute path name onto which the file system is to be mounted
+            - The path is case sensitive, and < 1023 characters long
+        type: str
+        required: True
+    src:
+        description:
+            - The file in zos that is to be mounted
+        type: str
+        required: True
+    fstype:
+        description:
+            - The type of file system that will perform the logical mount request.
+            - The system converts the TYPE operand value to uppercase letters.
+            - The value can be up to 8 characters long.
+        type: str
+        choices:
+            - HFS
+            - zFS
+            - NFS
+            - TFS
+        required: True
+    state:
+        description:
+            - The desired status of the described mount (choice).
+            - If I(absent), this will un-mount and alse remove from fstab if provided.
+            - If I(mounted), this will mount and add to the fstab if provided.
+            - >
+                If I(mounted) and mount source is in use, left in place.
+                Will also make sure entry is in fstab, if provided.
+            - >
+                If I(unmounted) and mount source is in use, device will be unmounted.
+                fstab is not altered, even when provided.
+                Module completes successfully with I(changed=True).
+            - >
+                If I(unmounted) and mount source is not in use, no action taken.
+                Module completes successfully with I(changed=False).
+            - >
+                If I(present), if makes sure device is in fstab, if provided.
+                Returns I(changed) if fstab was rewritten.
+            - >
+                If I(remounted) the device is unmounted and mounted again.
+                Always returns I(changed=True).
+        type: str
+        choices:
+            - absent
+            - mounted
+            - unmounted
+            - present
+            - remounted
+        required: False
+        default: mounted
+    fstab:
+        description:
+            - If provided, this is the full qualified name of the dataset member to change
+        type: str
+        required: False
+        aliases:
+            - bpxfile
+            - bpxprm
+    backup:
```

Comment:
We may want to verify backup behaves the same across modules, I believe other modules, such as zos_encode, may be good reference

Reply:
I had a meeting with Behnam which clarified this, and his comment strategy.  I've switched to backup=bool + backup_name=some pds entry for this, and am now supporting multiline comments with the agreed-on header/footer